styles/styles.css

```diff
@@ -0,0 +1,218 @@
+body {
+  font-family: sans-serif;
+}
+
+header {
+  background: url("../images/header-image.jpg") no-repeat top;
+  color: #81B741;
+  font-size: 1.25rem;
+  height: 20rem;
+  padding-top: 3rem;
+  box-sizing: border-box;
+}
+
+header img.logo {
+  width: 100%;
+}
+
+header .cart, .logo-container {
+  display: inline-block;
+
+}
+
+header .logo, .motto {
+  padding-left: 2rem;
+}
+
+@media (min-width: 840px) {
+  header .logo, .motto {
+    padding-left: 10rem;
+  }
+}
+
+.cart {
+  float: right;
+  padding: 1.875rem;
+}
+
+.cart img {
+  height: 1.25rem;
+}
+
+.cart span {
+  color: white;
+  font-size: .875rem;
+}
+
+.cart button {
+  background-color: #81B741;
+  color: #FFF;
+  display: block;
+}
+
+h1 {
+  font-size: 1.5625rem;
+}
+
+nav.global {
+  background-color: #E8E8E8;
+  color: #323232;
+  padding: 30px;
+  font-weight: bold;
+  display: flex;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+}
+
+nav a {
+  color: #323232;
+  text-decoration: none;
+  padding: 0 .9375rem;
+}
+
+a {
+  color: #49B0CE;
+  text-decoration: none;
+}
+
+main.container {
+  max-width: 900px;
+  margin: auto;
+}
+
+.intro-image {
+  display: inline-block;
+  float: left;
+  padding: 0 1.5625rem 1.5625rem 0;
+  height: 15.625rem;
+}
+
+.see-more {
+  margin-bottom: 2rem;
+  text-align: center;
+}
+
+.see-more form {
+  display: inline-block;
+}
+
+footer {
+  clear: both;
+  text-align: center;
+  background-color: #083B5F;
+  color: white;
+  padding: 3.125rem;
+}
+
+footer nav {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+footer nav a {
+  color: white;
+  text-decoration: none;
+  padding: 0 .9375rem;
+}
+
+footer img.logo {
+  width: 30%;
+}
+
+/* Dog Card Styles */
+.dog-cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  clear: both;
+}
+
+.dog-cards .card {
+  width: 10rem;
+  border: 1px solid #E8E8E8;
+  padding: .9375rem;
+  margin-bottom: .9375rem;
+  text-align: center;
+}
+
+.dog-cards img {
+  height: 4.6875rem;
```

Comment:
One thing you could do, is set the width to a certain percentage (cough, cough, 100%) and the height will adjust to maintain the aspect ratio.

Reply:
Oh thank you! I'll do that!